dev/package-examples/auditd-2.0.4/manifest.yml

```diff
@@ -1,3 +1,5 @@
+package_version: 1.0.0
```

Comment:
why not using just version? Like we do with `name` instead of `package_name`

Reply:
version is already taken: https://github.com/elastic/package-registry/pull/147/files/4eb902e5e7f2a83334bb2d8d54b76de823d929b0#diff-a2506afb478bf1deffc60f4eb9b4ea26R9

There are 2 versions:
* Version of the package, incremented with each addition
* Version of the package structure, incremented only when there is a change in the package structure (rarely hopefully). So far in Beats we never had to change it: https://github.com/elastic/beats/blob/master/filebeat/module/mongodb/log/manifest.yml#L1

Reply:
ah gotcha :+1:

Reply:
well, I'm pretty sure that confusion will pop up again ;-) Suggestion for a better more intuitive name?

Reply:
@exekias WDYT about `package_format` or `package_format_version`?

Reply:
both look good to me, also `format_version` as package is implicit

Reply:
+1 on `format_version`, will go with this one.